src/NUnitConsole/nunit3-console.tests/nunit3-console.tests.csproj

```diff
@@ -32,6 +32,8 @@
     <ProjectReference Include="..\..\NUnitEngine\nunit.engine\nunit.engine.csproj" />
     <ProjectReference Include="..\nunit3-console\nunit3-console.csproj" />
     <ProjectReference Include="..\..\NUnitEngine\nunit.engine.api\nunit.engine.api.csproj" />
+    <ProjectReference Include="..\..\NUnitEngine\nunit-agent\nunit-agent.csproj" />
+    <ProjectReference Include="..\..\NUnitEngine\nunit-agent\nunit-agent-x86.csproj" />
```

Comment:
Should you add the `<ReferenceOutputAssembly>false</ReferenceOutputAssembly>` from the SO post to these? They aren't directly referenced.